lib/minitest/assertions.rb

```diff
@@ -500,7 +500,7 @@ def pass _msg = nil
     # Fails if +test+ is truthy.
 
     def refute test, msg = nil
-      msg ||= "Failed refutation, no message given"
+      msg ||= "Expected #{mu_pp(test)} to not be truthy"
```

Comment:
hrm. as soon as you add interpolation, you should wrap it in a `message { ... }` block.